install_config/router/default_haproxy_router.adoc

```diff
@@ -166,6 +168,50 @@ $ oc adm router --dry-run -o yaml --service-account=router
 ----
 endif::[]
 
+[[deploy-router-access-logs]]
+Configuring the Router to Forward Access Logs::
+
+// tag::accesslogs[]
+You can configure {product-title} to send socket messages, or access logs, 
+to your EFK stack. The `--extended-logging` parameter appends an rsyslog 
+sidecar container to forward audit logs to standard output. 
+
+----
+$ oc adm router --extended-logging=true
+----
+
+For example, the following is a router configuration created using `--extended-logging=true`: 
+----
+# oc get pod test-router-1-xhdb9 -o yaml
+apiVersion: v1
+kind: Pod
+spec:
+  containers:
+  - env:
+    ....
```

Comment:
And the below env is added to router container with option `--extended-logging=true`, so better to add it here.
```
    - name: ROUTER_SYSLOG_ADDRESS
      value: /var/lib/rsyslog/rsyslog.sock
```

Reply:
Outdated